### ISSUE TITLE
[Accessibility : Keyboard Navigation] Unable to select "date" from keyboard

### ISSUE THREAD

**Issue by Nagamani71** [kimchi-project/wok#96]
![accessibility14](https://cloud.githubusercontent.com/assets/17590096/14138068/ba16e1b4-f688-11e5-9590-5d886af60224.PNG)
 When we click enter it is taking today's date in to that field and it is allowing to edit that field manually , but it is not allowing to select different date from calender through keyboard

**Comment by ghidasy** [comment]
I can confirm this issue, the calendar even updates itself when a date is inserted manually. 

The calendar is also navigable by the keyboard-only users if they use keyboard helpers.
